pkg/checks/dns/dns_test.go

```diff
@@ -190,13 +192,13 @@ func TestDNS_Run(t *testing.T) {
 				}
 			}()
 
-			r := <-c.ResultChan()
-			if r.Err != tt.want.Err {
-				t.Errorf("DNS.Run() = %v, want %v", r.Err, tt.want.Err)
+			r := <-cResult
+			if r.Result.Err != tt.want.Err {
+				t.Errorf("DNS.Run() = %v, want %v", r.Result.Err, tt.want.Err)
 			}
```

Comment:
We have no case where an error is present in the result struct. Is this an omission on our part or does the `Result` not a good match for the DNS check? The Result.err is always empty in the `Run` function of the DNS check...

Reply:
Wanna remove it now? I don't feel like we should open an issue for this.

Reply:
Sure, I've removed it from the struct and all tests. 😄